content/en-bulletin.md

```diff
@@ -12,21 +12,21 @@ _High school students from Itabo y San Gregorio de Nigua selected for the FIRST
 - Advance engineering education in K-12
 - Increase learning and understanding.
 
-## MACILE 2007 Summer Academy
+## MACILE Summer Academy 2007
 
 Forty-seven students completed the 2007 MACILE Summer Academy. It lasted for three weeks and only included mathematics. Students attended from 8:00 AM to 12:00 PM, Monday through Friday. We provided a healthy breakfast snack. At the end of the program, we celebrated with a barbeque.
 
-_A group of 7th-8th grade students from the MACILE Summer Academy 2007 at the barbeque. Students came from schools in Itabo, Nigua, El 18, Yogo Yogo, and Haina. Yahaira Pinales (center right), Dominican teacher that recruited the students (top right)._
+_A group of 7th-8th grade students from the MACILE Summer Academy 2007 at the barbeque. Students came from schools in Itabo, Nigua, El 18, Yogo Yogo, and Haina. Yahaira Soriano (center right), Dominican teacher who recruited the students (top right)._
 
 ### People that made a difference
 
-Local educators helped us out by visiting the schools and helping select potential students. They provided MACILE space and also assisted with logistics.
+Local educators and community leaders made the success of MACILE possible. They visited the schools to promote MACILE and encourage students participation. They provided the space for the program and assisted with logistics. 
```

Comment:
> Local educators and community leaders made the success of MACILE possible. They visited the schools to promote MACILE and encourage students participation. They provided the space for the program and assisted with logistics.

I would correct it as follows. The first sentece I rewrote. The rest I corrected corrected tense. 

Local educators and community leaders made the 2007 MACILE Summer Academy possible and successful. They visited the schools to promote MACILE and encouraged students participation. They provided the space for the program and assisted with logistics.